packages/compile-solidity/src/compilerSupplier/loadingStrategies/Native.ts

```diff
@@ -5,7 +5,7 @@ const { NoVersionError } = require("../errors");
 export class Native {
   load() {
     const versionString = this.validateAndGetSolcVersion();
-    const command = "solc --standard-json";
+    const command = (process.env.TRUFFLE_NATIVE_SOLC_PATH || "solc") + " --standard-json";
```

Comment:
This is nitpicking, but if we're going to argue about `||` vs `??`, I'd kind of prefer `||` because what if someone sets this variable to the empty string as a way of attempting to unset it, or something?  Idk exactly what the interface is on `process.env`, maybe this is a non-concern, but to my mind unless there's some particular falsy value you're worried about causing problems, it makes sense ot use `||` instead of `??`.

Reply:
I think you are right @haltman-at , that value can sometimes actually be empty string.

Reply:
I merged the suggestion but still using "||"

Reply:
Hey @hellwolf , sorry about the confusion. I requested changes for an issue I didn't document. I was tired and thought I'd comeback to it this morning, only to realize I submitted it. :disappointed:  I updated my review.

Reply:
No worries.

But I didn't know "nativePath" was an option, I don't seem to be able to find the reference to that in code.

Reply:
> But I didn't know "nativePath" was an option, I don't seem to be able to find the reference to that in code.

`nativePath` doesn't currently exist. However, the location I shared, is where one could define this new configuration, and set its defaults depending on the ENV. Of course, the setting will have to be consumed in [loadingStrategies/Native.ts](packages/compile-solidity/src/compilerSupplier/loadingStrategies/Native.ts), and I still think there's a better name than `nativePath`.